cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -171,6 +174,7 @@ func (s *CMServer) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&s.VolumeConfigFlags.PersistentVolumeRecyclerPodTemplateFilePathHostPath, "pv-recycler-pod-template-filepath-hostpath", s.VolumeConfigFlags.PersistentVolumeRecyclerPodTemplateFilePathHostPath, "The file path to a pod definition used as a template for HostPath persistent volume recycling. This is for development and testing only and will not work in a multi-node cluster.")
 	fs.IntVar(&s.VolumeConfigFlags.PersistentVolumeRecyclerMinimumTimeoutHostPath, "pv-recycler-minimum-timeout-hostpath", s.VolumeConfigFlags.PersistentVolumeRecyclerMinimumTimeoutHostPath, "The minimum ActiveDeadlineSeconds to use for a HostPath Recycler pod.  This is for development and testing only and will not work in a multi-node cluster.")
 	fs.IntVar(&s.VolumeConfigFlags.PersistentVolumeRecyclerIncrementTimeoutHostPath, "pv-recycler-timeout-increment-hostpath", s.VolumeConfigFlags.PersistentVolumeRecyclerIncrementTimeoutHostPath, "the increment of time added per Gi to ActiveDeadlineSeconds for a HostPath scrubber pod.  This is for development and testing only and will not work in a multi-node cluster.")
+	fs.StringSliceVar(&s.VolumeConfigFlags.ProvisionerQualityOfServiceTiers, "qos-tier", s.VolumeConfigFlags.ProvisionerQualityOfServiceTiers, "List of experimental quality-of-service tiers and their provisioners.  Values are QoSName/volume-plugin-name.")
```

Comment:
I will change it to `volume-qos-tier`.  Thanks for pointing out the overloaded term. 

In this case, tiers will eventually grow so that an admin can specify an SSD v. HD, higher IOPS for some volumes and greater throughput, glacial storage (in EBS), etc.